terraform/modules/ecr_migration/main.tf

```diff
@@ -9,6 +9,7 @@ terraform {
   }
 }
 
+#tfsec:ignore:aws-ecr-enforce-immutable-repository:mutable images preferred

```

Comment:
This looks like a duplicate of two lines below?

Reply:
Whoops good spot! I think I missed this when I merged the latest changes in from main (about s3-migration). Removed now 😀